src/lightning_app/cli/component-template/.github/workflows/ci-testing.yml

```diff
@@ -15,8 +15,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-20.04, macOS-10.15, windows-2019]
-        python-version: [3.8]
+        os: [ubuntu-20.04, macOS-11, windows-2019]
+        python-version: [3.7]
```

Comment:
```suggestion
        python-version: [3.8]
```

Reply:
Is this critical? I may keep it rather aligned with future/master branch...

Reply:
not critical